src/settings/Settings.ts

```diff
@@ -176,6 +176,12 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
         default: false,
     },
+    "feature_show_shared_rooms": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
```

Comment:
all the other labs settings seem to be on this level:

```suggestion
        supportedLevels: LEVELS_FEATURE,
```